.travis.rosinstall

```diff
@@ -1,3 +1,6 @@
 - git:
     uri: https://github.com/ros/std_msgs.git
     local-name: ros/std_msgs
+- git:
+    uri: https://github.com/ros-industrial/industrial_ci.git
```

Comment:
This job is using the file from the master branch.

It gets skipped in 
https://travis-ci.org/ros-industrial/industrial_ci/jobs/209393509
https://travis-ci.org/ros-industrial/industrial_ci/jobs/209393512
https://travis-ci.org/ros-industrial/industrial_ci/jobs/209393523